pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -132,7 +132,7 @@ self: super: builtins.intersectAttrs super {
     then let inherit (pkgs.darwin) security_tool;
       in pkgs.lib.overrideDerivation (addBuildDepend super.x509-system security_tool) (drv: {
         postPatch = (drv.postPatch or "") + ''
-          substituteInPlace System/X509/MacOS.hs --replace security ${security_tool}/bin/security
+          substituteInPlace System/X509/MacOS.hs --replace security /usr/bin/security
```

Comment:
could you add a FIXME comment or something

Reply:
yeah - added